website/src/segment.tsx

```diff
@@ -180,6 +184,13 @@ const WithTooltip = (p: {
   </Tooltip>
 )
 
+const FillerLine = () => (
+  <div className={css.lineBox}>
+    {" "}
+    <hr className={css.fillerLine} />{" "}
```

Comment:
Very last thing, we can get rid of these `{" "}` which are just explicit spaces that got added when reformatting, because we don't need any explicit extra space characters here.